examples/expo/App.js

```diff
@@ -78,6 +80,12 @@ export default () => {
         }}
       />
       <AlwaysOpen />
+
```

Comment:
Last comment, I don't think you need a specific example for it. I think you can just add it to a current example since it's a simple props. Maybe on the section list or flat list, having a floating button scroll top/bottom depending on where you are, would be nice and enough!